core/graph/src/test/java/uk/gov/gchq/gaffer/graph/GraphTest.java

```diff
@@ -1844,54 +1948,6 @@ public void shouldThrowExceptionOnExecuteJobWithANullContext() throws OperationE
         }
     }
 
-    @Test
-    public void shouldThrowExceptionOnExecuteWithANullUserInContext() throws OperationException {
```

Comment:
These tests have been removed as a context can no longer be created without a user.